build.sbt

```diff
@@ -74,24 +67,12 @@ def coreProject(scalaCheckVersion: String): Project = {
 
 lazy val `core_1-12` = coreProject(Dependencies.scalaCheck12Version)
 lazy val `core_1-13` = coreProject(Dependencies.scalaCheck13Version)
+lazy val `core_1-14` = coreProject(Dependencies.scalaCheck14Version)
```

Comment:
These definitions, and the ones for joda time below, seem like DRY violations.

Reply:
How so? Each `lazy val` is required to make it a project, and I've boiled the project down to a single variable: the version of scalacheck.

Reply:
I just saw repeated variables that all seem to depend on `Dependencies.scalaCheck__Version` and that makes me think "Use the [loops], Luke!"